ansible.cfg

```diff
@@ -0,0 +1,6 @@
+[defaults]
+library = foreman-ansible-modules/modules
+module_utils = foreman-ansible-modules/module_utils
+roles_path = $PWD/galaxy_roles:$PWD/roles
+inventory = inventory
+retry_files_enabled = False
```

Comment:
A few others to consider:

```
force_color = 1
stdout_callback = yaml
display_skip_hosts = false
```

Reply:
Ok, will take a look at each of those options and will add it up.